modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -967,10 +943,14 @@ impl Wallet {
             return Err(ProcessPegOutSigError::ErrorFinalizingPsbt(error));
         }
 
-        let tx = unsigned.psbt.clone().extract_tx();
+        let tx = unsigned
+            .psbt
+            .clone()
+            .extract_tx()
+            .map_err(ProcessPegOutSigError::ErrorExtractingTx)?;
```

Comment:
This needs to be an expect! You can not introduce new error here.

Reply:
Ahh nice catch! You're right, the source code appears to be the same as `extract_tx` on bitcoin v0.32:

https://docs.rs/bitcoin/0.30.2/src/bitcoin/psbt/mod.rs.html#131-140
https://docs.rs/bitcoin/0.32.4/src/bitcoin/psbt/mod.rs.html#172-184